packages/react/src/components/ChoiceGroup/ChoiceGroup.test.tsx

```diff
@@ -207,80 +186,74 @@ describe('ChoiceGroup', () => {
     const choiceGroupRef = React.createRef<IChoiceGroup>();
-    choiceGroup = mount(<ChoiceGroup options={TEST_OPTIONS} componentRef={choiceGroupRef} />);
+    const { getAllByRole } = render(<ChoiceGroup options={TEST_OPTIONS} componentRef={choiceGroupRef} />);
 
-    const choiceOptions = choiceGroup
-      .getDOMNode()
-      .querySelectorAll(CHOICE_QUERY_SELECTOR) as NodeListOf<HTMLInputElement>;
+    const choiceOptions = getAllByRole('radio') as HTMLInputElement[];
 
     expect(choiceGroupRef.current!.checkedOption).toBeUndefined();
-    ReactTestUtils.Simulate.change(choiceOptions[0]);
+    userEvent.click(choiceOptions[0]);
     expect(choiceGroupRef.current!.checkedOption).toEqual(TEST_OPTIONS[0]);
   });
 
   it('returns the current checked option with defaultSelectedKey', () => {
     const choiceGroupRef = React.createRef<IChoiceGroup>();
-    choiceGroup = mount(<ChoiceGroup options={TEST_OPTIONS} defaultSelectedKey="1" componentRef={choiceGroupRef} />);
+    const { getAllByRole } = render(
+      <ChoiceGroup options={TEST_OPTIONS} defaultSelectedKey="1" componentRef={choiceGroupRef} />,
+    );
 
-    const choiceOptions = choiceGroup
-      .getDOMNode()
-      .querySelectorAll(CHOICE_QUERY_SELECTOR) as NodeListOf<HTMLInputElement>;
+    const choiceOptions = getAllByRole('radio') as HTMLInputElement[];
 
     expect(choiceGroupRef.current!.checkedOption).toEqual(TEST_OPTIONS[0]);
-    ReactTestUtils.Simulate.change(choiceOptions[1]);
+    userEvent.click(choiceOptions[1]);
     expect(choiceGroupRef.current!.checkedOption).toEqual(TEST_OPTIONS[1]);
   });
 
   it('returns the current checked option with selectedKey', () => {
     const choiceGroupRef = React.createRef<IChoiceGroup>();
-    choiceGroup = mount(<ChoiceGroup options={TEST_OPTIONS} selectedKey="1" componentRef={choiceGroupRef} />);
+    const { getAllByRole } = render(
+      <ChoiceGroup options={TEST_OPTIONS} selectedKey="1" componentRef={choiceGroupRef} />,
+    );
 
-    const choiceOptions = choiceGroup
-      .getDOMNode()
-      .querySelectorAll(CHOICE_QUERY_SELECTOR) as NodeListOf<HTMLInputElement>;
+    const choiceOptions = getAllByRole('radio') as HTMLInputElement[];
 
     expect(choiceGroupRef.current!.checkedOption).toEqual(TEST_OPTIONS[0]);
-    ReactTestUtils.Simulate.change(choiceOptions[1]);
+    userEvent.click(choiceOptions[1]);
     // selectedKey is still used even though it didn't get updated for latest user click
     expect(choiceGroupRef.current!.checkedOption).toEqual(TEST_OPTIONS[0]);
   });
 
   it('can render element id', () => {
-    choiceGroup = mount(<ChoiceGroup defaultSelectedKey="1" id="foo" options={TEST_OPTIONS} />);
-    expect(choiceGroup.getDOMNode().getAttribute('id')).toBe('foo');
+    const { container } = render(<ChoiceGroup defaultSelectedKey="1" id="foo" options={TEST_OPTIONS} />);
+    const root = container.querySelector('.ms-ChoiceFieldGroup');
```

Comment:
Is there a reasonable way to get the root element by role? Or if it should be the actual DOM root, it might be better to do `container.firstElementChild`.

Reply:
went with `container.firstElementChild` approach here